WorkflowFunctionAdminBundle/DependencyInjection/Compiler/RoleCompilerPass.php

```diff
@@ -26,5 +26,13 @@ public function process(ContainerBuilder $container)
             WorkflowFunctionPanelStrategy::ROLE_ACCESS_UPDATE_WORKFLOWFUNCTION,
             WorkflowFunctionPanelStrategy::ROLE_ACCESS_DELETE_WORKFLOWFUNCTION,
         ));
+
+        if ($container->hasParameter('open_orchestra_backoffice.role')) {
+            $param = $container->getParameter('open_orchestra_backoffice.role');
```

Comment:
Instead of ``$param`` can you find something more explicit like ``roles`` or ``rolesList`` or something like this ?